web/html/src/manager/content-management/shared/components/panels/promote/promote.js

```diff
@@ -75,7 +74,11 @@ const Promote = (props: Props) => {
               <dl className="row">
                 <dt className="col-xs-4">{t('Version')}:</dt>
                 <dd className="col-xs-8">
-                  <pre>{versionMessage}</pre>
+                  <BuildVersion
```

Comment:
We have the same in build/build.js but it seems this was not changed. Please check

Reply:
I did it on purpose because in the `build` popup form we don't have yet the "commit message" for the version that it is supposed to handle the click and hide/show details. In there I think we should always show the entire version detail expanded. It was like that before, and I felt it is still a valid behavior for that specific case. What do you think?

Reply:
Ok, fine with me. I just thought you forgot it.